### ISSUE TITLE
good start

### ISSUE THREAD

**Issue by stuydw** [hunter-teacher-cert/currdev_unit_plan-jsdom#1]
Javascript DOM manipulation can be a lot of fun. A few thoughts that I have:
Will you be using the web browser console to do some of the development/testing? I've had success in class doing things like loading an ESPN page that shows "Mr. DW Hits Game Winning Home Run", etc.
What kind of development environment will the students be working in?
What will you be using to host the student's web pages?

**Comment by brianmueller** [comment]
Yes, we will definitely use the browser console! It's very helpful for debugging, testing your code along the way, seeing the live status of the DOM in the elements tab, etc.
The students will be using the CS50 IDE from Harvard, built on top of Cloud9. It's much closer to a local IDE than something like repl.it. Students use the command line, setup an SSH key, use git commands, preview their websites locally, push when ready, etc. All of that material is learned the year before.
We can push to Github for use of Github pages.
